### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by GitHubEmploy**
This adds support for installing on 24.10 ubuntu systems

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,6 +66,18 @@ sudo dpkg -i betaflight-configurator_10.10.0_amd64.deb
 sudo apt-get -f install
 ```
 
+On Ubuntu 24.10 and above, please follow these steps, as some deprecated modules are no longer avaliable through apt on this distro:
```
Comment (haslinghuis) [inline README.md:69]:
```suggestion
On Ubuntu 24.10 and above, please follow these steps, as some deprecated modules are no longer available through apt on this distro:
```

File: README.md [hunk 2]
```diff
@@ -66,6 +66,18 @@ sudo dpkg -i betaflight-configurator_10.10.0_amd64.deb
...
+```
+sudo echo "deb http://archive.ubuntu.com/ubuntu/ lunar universe" > /etc/apt/sources.list.d/lunar-repos-old.list
```
Comment (nerdCopter) [inline README.md:71]: If you are downloading .deb files directly, why add any apt source? is it for dependencies?

Comment (GitHubEmploy) [reply README.md:71]: Looking back on it now, i actually don't think its required, but I did have that installed, (from trying to originally follow the other ubuntu instructions) and its very likely that some of the modules I installed manually have dependencies from the universe package.

I can test if those sources are needed in a VM if needed, but theres no harm to adding it anyhow

Comment (GitHubEmploy) [reply README.md:71]: Hello, 

I have rebuilt it in a VM of ubuntu 24.10, those dependencies are not strictly required

I can remove from the MR

**Review by haslinghuis — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**